library/src/main/java/com/devbrackets/android/exomedia/core/video/ExoVideoView.java

```diff
@@ -281,7 +281,9 @@ public void onSurfaceTextureAvailable(SurfaceTexture surfaceTexture, int width,
 
         @Override
         public boolean onSurfaceTextureDestroyed(SurfaceTexture surfaceTexture) {
+            Surface surface = emExoPlayer.getSurface();
```

Comment:
I thought this class is responsible to release the surface (also only this class calls setSurface() and blockingClearSurface()).
Anyway thanks for fix..!